stable/admin/values.yaml

```diff
@@ -223,3 +223,7 @@ admin:
 
   #Some clusters require longer readiness probe timeouts
   readinessTimeoutSeconds: 1
+
+  #These annotations will pass throgh to the pod as supplied, useful for integrating 3rd party products such as Vault.
+  podAnnotations: {}
```

Comment:
As the annotations go to the controller K8s resource (i.e statefulset or deployment) maybe the variable name could be something like `customAnnotations` or `userDefinedAnnotations` or just `annotations`. This is just a suggestion.

Reply:
I don't mind, but we need a way to seperate them from service annotations, for example... hence I moved away from just annotations...